page/about-jquery/using-jquery.md

```diff
@@ -0,0 +1,160 @@
+
+### Adding and Removing an HTML Class
+
+jQuery is often used to bind events to elements in the DOM, assigning them functionality to be triggered by a user's actions on the page. To demonstrate, let's highlight the menu item a user last clicked by changing its HTML class to "active", and assigning the "active" class a different appearance using CSS.
+
+First, copy or create the `index.html` file below:
+
+```
+// index.html
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Demo</title>
+</head>
+<body>
+  <nav>
+    <ul>
+      <li><a href="http://jquery.com">Home</a></li>
+      <li><a href="http://google.com">About</a></li>
+      <li><a href="http://github.com">Products</a></li>
+    </ul>
+  </nav>
+  <script src="jquery.js"></script>
+  <script src="demo.js"></script>
+  <script src="main.css"></script>
+</body>
+</html>
+```
+
+The `<script src>` tags reference the jQuery library, which you should have downloaded and copied into your directory, as well as the files `demo.js` and `main.css`. If you don't already have them, create these two files in the same directory and copy their content as shown below.
+
+* **Note:** Normally, these different types of files are sorted into folders that represent the overall structure of the webpage, reflecting the separation between content, action, and style, or, in parallel terms, model, controller, and view. Though you could alter the CSS of an element directly in the JavaScript by using jQuery's [`.css()`](http://api.jquery.com/css/) method, that piece of information about the style won't be included with the rest of the CSS, making it hard to find in the future, or for other developers. Similarly, you could construct HTML and text in a JavaScript file to be appended to the DOM, but again, that content won't be included with the rest of the HTML, where it belongs. Instead, organize and label your content in HTML, use JavaScript and jQuery to assign events to the elements, and let the CSS take care of the styling.
+
+```
+// demo.js
+$( document ).ready(function( event ) {
+
+  // Your code here
+
+  event.preventDefault();
+
+});
+```
+
+```
+// main.css
+a.active {
+  font-weight: bold;
+}
+```
+
+**Important:** *You must place the remaining jQuery examples inside the `ready` event so that your code executes when the document is ready to be worked on, as explained in the [previous setup tutorial](http://learn.jquery.com/about-jquery/how-to-setup-jQuery/).*
+
+Add the [.addClass()](http://api.jquery.com/addClass/) call inside the `ready` event:
+
+```
+$( "a" ).addClass( "active" );
+```
+
+All `<a>` elements are now bold. If you right click on a link and select "Inspect Element" you'll see that the class of "active" has been added to all `<a>` elements.
```

Comment:
Other browsers don't have this item in the context menu. This description should be expanded.